src/main/kotlin/no/nav/amt/arena/acl/repositories/ArenaDataRepository.kt

```diff
@@ -188,4 +190,34 @@ open class ArenaDataRepository(
 		return template.update(sql, EmptySqlParameterSource())
 	}
 
+	fun getReingestableDeltakerWithStatus(status: IngestStatus): List<ArenaDataDbo> {
+		return getReingestableDeltakerWithStatus(listOf(status));
+	}
+
+	private fun getReingestableDeltakerWithStatus(
+		statuses: List<IngestStatus>,
+		offset: Int = 0,
+		limit: Int = 1000
+	): List<ArenaDataDbo> {
+		val sql = """
+			SELECT *
+				FROM arena_data deltaker join arena_data tiltak on deltaker.after ->> 'TILTAKGJENNOMFORING_ID' = tiltak.arena_id
+				WHERE deltaker.arena_table_name = '$ARENA_DELTAKER_TABLE_NAME'
+				  AND tiltak.arena_table_name = '$ARENA_GJENNOMFORING_TABLE_NAME'
+				  AND deltaker.ingest_status = :ingestStatuses
```

Comment:
Vil dette fungere, må man ikke bruke `deltaker.ingest_status = in(:ingestStatuses)` eller noe lignende?

Reply:
Jo, det må man :)